src/main/java/seedu/fma/ui/ResultDisplay.java

```diff
@@ -25,4 +30,28 @@ public void setFeedbackToUser(String feedbackToUser) {
         resultDisplay.setText(feedbackToUser);
     }
 
+    public void setSuggestionList(List<String> commandMessageUsage) {
+        this.commandSuggestionList = commandMessageUsage;
+    }
+
+    /**
+     * Show autocomplete result to user.
+     * @param input input of the user
+     */
+    public void showAutoCompleteResult(String input) {
+        requireNonNull(input);
+
+        if (input.length() == 0) {
+            return;
+        }
+
+        String feedback = "";
+        input = input.length() > 3 ? input.substring(0, 3) : input;
```

Comment:
Could split input and use the first string
`input.split(' ')[0]`
-> checking the first 3 char substring may be brittle (e.g. may cause unforeseen bugs if we change the commands e.g. if got addelephant command, addex will show addex and addelephant)

Reply:
Resolved.